### PR TITLE
Transparent Feature Added

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#118 

#### What does this implement/fix? Explain your changes.
+ transparent background added
+ `set_background` function added

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- `transparent` bgcolor
```
Comment (sepandhaghighi) [inline CHANGELOG.md:9]: Move this line to `### Changed` section, something like:
- Transparent mode support for `bgcolor` parameter

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: samila/functions.py [hunk 2]
```diff
@@ -172,6 +174,27 @@ def select_color(color):
...
+def set_background(bgcolor, fig, ax):
+    """
+    Set background for figure and axis.
+
+    :param bgcolor: given background color
+    :type bgcolor: any format
+    :param fig: figure
+    :type fig: matplotlib.pyplot.figure
+    :param ax: axis
+    :type ax: matplotlib.pyplot.figure
```
Comment (sepandhaghighi) [inline samila/functions.py:186]: Is it correct?

File: README.md [hunk 3]
```diff
@@ -180,6 +180,7 @@ Samila is a generative art generator written in Python, Samila let's you create
     3. Hex (example: `color="#eeefff"`)
     4. Random (example: `color="random"`)
     5. Complement (example: `color="complement", bgcolor="blue"`)
+    6. Transparent (example: `bgcolor="transparent"`)
```
Comment (sepandhaghighi) [inline README.md:184]: Please update here with this sentence:

⚠️ **Transparent** mode is only available for background

File: examples/demo.ipynb [hunk 4]
```diff
@@ -161,7 +161,8 @@
     "    2. RGB/RGBA (example: `color=(0.1,0.1,0.1)`, `color=(0.1,0.1,0.1,0.1)`)\n",
     "    3. Hex (example: `color=\"#eeefff\"`)\n",
     "    4. Random (example: `color=\"random\"`)\n",
-    "    5. Complement (example: `color=\"complement\", bgcolor=\"blue\"`)"
+    "    5. Complement (example: `color=\"complement\", bgcolor=\"blue\"`)\n",
+    "    6. Transparent (example: `bgcolor=\"transparent\"`)"
    ]
```
Comment (sepandhaghighi) [inline examples/demo.ipynb:166]: Please update here with this sentence:

⚠️ **Transparent** mode is only available for background

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: samila/functions.py (hunk 2 above)
Comment (sadrasabouri) [reply samila/functions.py:186]: Solved in new commit.

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🚀

**Pull request merged**